CRM/Core/Page/File.php

```diff
@@ -33,17 +33,17 @@ public function run() {
     // File ID
     $fileId = CRM_Utils_Request::retrieve('id', 'Positive', $this, FALSE);
     $fileName = CRM_Utils_Request::retrieve('filename', 'String', $this, FALSE);
-    if (empty($fileName) && (empty($entityId) || empty($fileId))) {
-      CRM_Core_Error::statusBounce("Cannot access file: Must pass either \"Filename\" or the combination of \"Entity ID\" + \"File ID\"");
+    if (empty($fileName) && empty($fileId)) {
+      CRM_Core_Error::statusBounce('"Cannot access file: Must pass either "Filename" or the combination of "File ID" + "Hash""');
```

Comment:
> `'"Cannot ... ""'`

This seems to be surrounded by double quotes now, whereas before it was not. Is that intentional?

Reply:
@Sjord good catch, fixed.